jekyll/features.html

```diff
@@ -9,19 +9,32 @@ <h1 class="text-centered page-title main-heading">Features</h1>
   <section class="content">
     <div class="pure-g">
       <div class="pure-u-1">
-        <h1>Intuitive and clean syntax</h1>
+        <h1>One language to rule them all</h1>
       </div>
       <div class="pure-u-1 pure-u-md-1-2 code-align">
-        <h2>
-          Taking inspiration from multiple languages, the Nim syntax is
-          easily understood and results in code that can be modified very
-          effectively.
-        </h2>
+        <h2>Good for everything, from shell scripting to web front and backend,
+          to machine learning, high-performance computing (HPC), and embedded.</h2>
         <ul>
-          <li>Indentation significant syntax</li>
-          <li>Multiple constructs inspired by Python</li>
-          <li>Multi-line lambdas</li>
-          <li>It combines successful concepts from Ada and Modula.</li>
+          <li>Productivity: human-readable and expressive like Python</li>
+          <li>Type system: strong static typing, nice generics, concepts, and type classes</li>
+          <li>Performance: matches C and C++, always gives control to the programmer</li>
+          <li>Portability: compiles to C, C++, and Javascript
+            <ul>
+              <li>Native use of all libraries in target languages (no ABI issues) + really nice FFI</li>
+              <li>Single (tiny) executable with no dependencies</li>
+              <li>Write server/client in the same language and share code between front/back end</li>
```

Comment:
```suggestion
              <li>Write the whole stack in the same language and share code between frontends and backends</li>
```

"server/client" sounded to limiting, can be microservices, etc.

"frontends and backends" plural sounds more prepared for bigger projects.

Reply:
the whole stack is a bit confusing. What's the meanings of "the whole stack"? I cannot find many definitions of it on google.